irods/test/harness/docker_container_driver.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
```

Comment:
Put in a general comment as to what the script does, as this is an important one.  (It calculates the internal path for a given external one, and launches that script in a standalone Docker container.)